### ISSUE TITLE
bitmap doesn't work

### ISSUE THREAD

**Issue by Dario-Ciceri** [mrfaptastic/ESP32-HUB75-MatrixPanel-I2S-DMA#282]
I've followed instructions inside bitmap example but it doesn't work, could you help me?

**Comment by DarrylStrong** [comment]
Are you going to give us a clue as to what is or isn't happening?

**Comment by Dario-Ciceri** [comment]
> Are you going to give us a clue as to what is or isn't happening?

sorry, the tool works it was this line "PROGMEM uint8_t * const" which can't be used

**Comment by Dario-Ciceri** [comment]
> That's not very clear, you have an error while compiling?

yes the code want char []

**Comment by zesinger** [comment]
Could you just share the error?

**Comment by Dario-Ciceri** [comment]
> Could you just share the error?

Yeah I'm sorry I'm not at home simply the image converter generate PROGMEM uint8_t * const and not char[]

**Comment by mrfaptastic** [comment]
Compiler error output plz
